src/java.base/share/classes/java/io/PushbackInputStream.java

```diff
@@ -53,6 +54,8 @@
  * @since   1.0
  */
 public class PushbackInputStream extends FilterInputStream {
+
+    // initialized to null when BufferedInputStream is sub-classed
```

Comment:
Thanks for the update, I think the change to PBIS looks okay, it's just this comment, which looks like it was copied from BIS.

Reply:
Fixed. Thanks, I have overlooked this one!